keyboards/bt66tech/readme.md

```diff
@@ -0,0 +1,19 @@
+# bt66tech60
+
+![bt66tech60](https://i.imgur.com/8eJ4liU.jpg)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [bt66](https://github.com/bt66)
+* Hardware Supported: *handwired, STM32*
```

Comment:
If this is handwired, we generally prefer that it goes in the handwired folder, as that is the place for handwired boards, and passion projects/small run boards.